.changeset/neat-beds-unite.md

```diff
@@ -0,0 +1,22 @@
+---
+"remix": minor
+"@remix-run/cloudflare": minor
+"@remix-run/deno": minor
+"@remix-run/node": minor
+"@remix-run/react": minor
+"@remix-run/serve": minor
+"@remix-run/server-runtime": minor
+---
+
+feat(@remix-run/cloudflare,@remix-run/deno,@remix-run/node): `SerializeFrom` utility for loader and action type inference
```

Comment:
We don't need the conventional commit format here. Affected packages and change type are already included in the changesets.